qcodes/instrument/parameter.py

```diff
@@ -1545,3 +1547,197 @@ def __init__(self, name, instrument=None, initial_value=None, **kwargs):
         super().__init__(name=name, instrument=instrument,
                          get_cmd=None, set_cmd=None,
                          initial_value=initial_value, **kwargs)
+
+
+class ParameterScaler(Parameter):
```

Comment:
@YakBizzarro , sorry for the late comment, but I would like to rename this class to ScaledParameter. The reason is the following. A "scaler" (so far as I my knowledge of English is concerned) is smth that "scales" smth. According to the implementation, the internal "multiplier" parameter is doing the actual "scaling", while the "ParameterScaler" instance just returns the "scaled" value of the wrapped parameter. In other words, if I call `get` for the "ParameterScaler" I get the "scaled" value. Thus, "ParameterScaler" shall be called "ScaledParameter".

Reply:
Ok, seems legit